examples/docs_beta_snippets/docs_beta_snippets/dagster-plus/deployment/serverless/runtime-environment/github_disable_pex.yaml

```diff
@@ -0,0 +1,6 @@
+env:
+  DAGSTER_CLOUD_URL: "http://jamie-test-1.canary.dagster.cloud"
+  DAGSTER_CLOUD_API_TOKEN: ${{ secrets.DAGSTER_CLOUD_API_TOKEN }}
+  # highlight-start
+  # ENABLE_FAST_DEPLOYS: 'true'
```

Comment:
should this be set to false instead of commented out?

Reply:
yeah, i assume that would also work. I pulled this from the existing guide where it gets commented out. but i dont see why setting to false wouldn't work